Nie-ontologies/Generic-ontologies/scholarly-editing.ttl

```diff
@@ -189,14 +189,14 @@ scholarly-editing:Siglum
 scholarly-editing:Apparatus
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/scholarly-editing#>;
 	a rdfs:Class;
-	rdfs:label "editing apparatus"@en, "Edierungsapparatus"@de, "apparatus editionis"@la, "apparat éditionel"@fr, "apparato edizione"@it; 
+	rdfs:label "editing apparatus"@en, "textkritischer Apparat"@de, "apparatus editionis"@la, "apparat éditionel"@fr, "apparato edizione"@it; 
```

Comment:
I'll check on the other labels.

Reply:
Changed it to "editorischer Apparat" because later critical apparatus = textkritischer Apparat. Maybe we discuss that on Friday.
See Commit 9198738